scripts/defaults-rhel10.sh

```diff
@@ -6,4 +6,5 @@ source ./network-device-names.cfg
 export KSTEST_URL='http://download.devel.redhat.com/rhel-10/nightly/RHEL-10/latest-RHEL-10.1/compose/BaseOS/x86_64/os/'
 export KSTEST_MODULAR_URL='http://download.devel.redhat.com/rhel-10/nightly/RHEL-10/latest-RHEL-10.1/compose/AppStream/x86_64/os/'
 export KSTEST_FTP_URL='ftp://download.devel.redhat.com/mnt/redhat/rhel-10/nightly/RHEL-10/latest-RHEL-10.1/compose/BaseOS/x86_64/os/'
+export KSTEST_FTP_APPSTREAM_URL='ftp://download.devel.redhat.com/mnt/redhat/rhel-10/nightly/RHEL-10/latest-RHEL-10.1/compose/AppStream/x86_64/os/'
```

Comment:
Perhaps the variable could be defined as
`KSTEST_FTP_APPSTREAM_URL="${KSTEST_FTP_URL/BaseOS/AppStream}"`
to reduce the number of places where the release version has to be changed for every minor release.

Or even better, the RHEL release version could be specified as a new variable only once in the file and substituted in all the URL variables. But that's for a separate PR. What do you think?

Reply:
I would say it is a good idea, however, I think it should be a separate PR focused on this change.